charts/workflows/charts/move2kube/INSTALL.md

```diff
@@ -0,0 +1,55 @@
+Move2kube
```

Comment:
You are missing the README.md from the origin repostiory

Reply:
you mean I did not have a README file for the move2kube chart?
Otherwise why would I need the README from the orchestrator helm chart?

Reply:
My intention was to include the generated README.md file from this closed PR: https://github.com/parodos-dev/orchestrator-helm-chart/pull/41/files#diff-bb77b2ae94397ff4554248dcc1b7d3cd70f2aaf63e62d7770c1fefdba90b49e2

but I can do it after this PR is merged.

Reply:
Actually, now this PR includes both the INSTALL.md and the README.md, so we can resolve this comment as fixed.